src/client/sandbox/cookie/server-sync.js

```diff
@@ -0,0 +1,122 @@
+import nativeMethods from '../../sandbox/native-methods';
+import XhrSandbox from '../../sandbox/xhr';
+import settings from '../../settings';
+import { stringify as stringifyJSON } from '../../json';
+import { isIE9 } from '../../utils/browser';
+
+const SWITCH_BACK_TO_ASYNC_XHR_DELAY = 2000;
+
+export default class ServerSync {
+    constructor () {
+        this.useAsyncXhr = true;
+        this.queue       = [];
+        this.sendedQueue = null;
```

Comment:
But we don't change this array, we just assign him [another array](https://github.com/DevExpress/testcafe-hammerhead/pull/1128/files#diff-e693d417f378fee695a11ea6f7e61f78R78).
Therefore, it just create unnecessary empty array if we initialize it as `this.sendedQueue = []`.